src/stylesheets/core/_iae-global.scss

```diff
@@ -0,0 +1,130 @@
+// Globals
+//
+// Global declarative helper classes.
+//
+// Styleguide globals.global
+
+.clearfix:after {
+     visibility: hidden;
+     display: block;
+     font-size: 0;
+     content: " ";
+     clear: both;
+     height: 0;
+     }
+    
+.disable {
+    color: $color-gray-warm-dark;
+    cursor: default;
+    &:hover {
+        background-color: transparent !important;
+        color: $color-gray-warm-dark !important;
+    }
+    &:visited{
+        color: $color-gray-warm-dark !important;
+    }
+}
+
+.sr-only {
+    position: absolute;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
+    height: 1px;
+    width: 1px;
+    margin: -1px;
+    padding: 0;
+    border: 0;
+}
+
+.show {
+    display: block;
+}
+
+.hide {
+    display:none;
+}
+
+.inline {
+    display: inline;
+}
+
+.relative {
+    position: relative
+}
+
+.width100 {
+    width: 100%;
+}
+
+.middle-align {
+    display: table-cell;
+    vertical-align: middle
+}
+
+@mixin transition($args...) {
+    -webkit-transition: $args;
+    -moz-transition: $args;
+    -ms-transition: $args;
+    -o-transition: $args;
+    transition: $args;
+}
+
```

Comment:
Can we move this @mixin to `/src/stylesheets/_utilities.scss`?

Curious: Does this duplicate the Bourbon method? http://bourbon.io/docs/#transitions